include/utils.php

```diff
@@ -279,6 +299,26 @@ function get_sugar_config_defaults(): array
 
     $sugar_config_defaults = [
         'admin_export_only' => false,
+        'anti_malware_scanners' => array(
```

Comment:
Short array syntax is preferred :).

```suggestion
        'anti_malware_scanners' => [
```